test/fixtures/definitions.txt

```diff
@@ -83,5 +153,176 @@ text with **inlines**, \[backslash escapes\], typography(tm), "smart quotes", ch
     text with **inlines**, \[backslash escapes\], typography(tm), "smart quotes", character references (&#x1f600;) &amp;&nbsp;entities.
 .
 <p>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</p>
-<blockquote>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</blockquote>
+<blockquote>
+<p>text with <strong>inlines</strong>, [backslash escapes], typography™, “smart quotes”, character references (😀) &amp; entities.</p>
+</blockquote>
+.
+
+Thematic breaks and headings should be rendered within indented quotes
+.
+    * * *
+
+    ---
+
+    # ATX Heading
+
+    SEText Heading
+    --------------
+.
+<blockquote><hr /></blockquote>
+<blockquote><hr /></blockquote>
+<blockquote><h1>ATX Heading</h1></blockquote>
+<blockquote><h2>SEText Heading</h2></blockquote>
+.
+
+Indented quotes separated by blank indented lines (spaces or tabs) are part of the same block quote.
+.
+    chunk1
+    
+      chunk2
+.
+<blockquote>
+<p>chunk1</p>
+<p>chunk2</p>
+</blockquote>
+.
+
+Indented quotes are rendered as `q.blockquote` elements when not separated from the previous paragraph.
+.
+Foo
+    bar
+.
+<p>Foo
+<q class="blockquote">bar</q></p>
+.
+
+Indented quotes are rendered as `q.blockquote` elements when not separated from the subsequent paragraph.
+.
+    foo
+bar
+.
+<p><q class="blockquote">foo</q>
+bar</p>
+.
+
+Indented quotes are rendered as `q.blockquote` elements when they are inside a paragraph with no separation.
+.
+Paragraph text:
+    Indented quote.
+more paragraph text.
 .
+<p>Paragraph text:
+<q class="blockquote">Indented quote.</q>
+more paragraph text.</p>
+.
+
+Blank indented lines around a quote within a paragraph are not rendered.
```

Comment:
The blank lines in this case are indented, which is how one can put a block quote inside a paragraph but separate it in the Markdown.